README.md

```diff
@@ -7,26 +7,26 @@ process easy and effective for everyone involved.
 
 1. Install Wordpress >= 3.8.1
   * Must be in the document root of the webserver (otherwise images won't load :( )
-  * Enter what you like for site title, admin user and password, none of this is used by git
+  * Enter what you like for site title, admin user and password, none of this is stored in git
 2. Clone the www repo into your wp-content/themes folder under the name 'owncloudorgnew'
```

Comment:
the easiest way is to clone the website wherer you like and have permissions to change the files and then create a link to it in the wp-content/themes in your wordpress instalation.